src/script/api/script_bridge.hpp

```diff
@@ -66,10 +66,12 @@ class ScriptBridge : public ScriptObject {
 	/**
 	 * Get the name of a bridge.
 	 * @param bridge_id The bridge to get the name of.
+	 * @param vehicle_type The vehicle-type of bridge to get the name of.
 	 * @pre IsValidBridge(bridge_id).
+	 * @pre vehicle_type == ScriptVehicle::VT_ROAD || vehicle_type == ScriptVehicle::VT_RAIL || vehicle_type == ScriptVehicle::VT_WATER
 	 * @return The name the bridge has.
 	 */
-	static char *GetName(BridgeID bridge_id);
+	static char *GetName(BridgeID bridge_id, ScriptVehicle::VehicleType vehicle_type);
```

Comment:
I think it would be better if this was a default parameter (to VT_RAIL)
I've no idea if this works with squirrel (seems like squirrel itself can have default parameters), but it'd let you remove all the dirty compatibility code

Reply:
Update: frosch says it would not :(